Ical.Net.nuspec

```diff
@@ -2,16 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
         <id>Ical.Net</id>
-        <version>2.2.5</version>
+        <version>2.1.5</version>
         <title>Ical.Net (formerly dday.ical)</title>
         <authors>Rian Stockbower, Douglas Day, M. David Peterson</authors>
         <owners>Rian Stockbower</owners>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>An iCalendar (RFC 5545) control library for .NET. A fork and continuation of the original dday.ical library including bug fixes, and many performance enhancements.</description>
         <projectUrl>https://github.com/rianjs/ical.net</projectUrl>
         <licenseUrl>https://github.com/rianjs/ical.net/blob/master/license.md</licenseUrl>
-        <iconUrl>https://github.com/rianjs/ical.net/raw/master/logo.png</iconUrl>
```

Comment:
Thanks for your patience & review as I get my head around Github – While I have used it in the past, it has only been for projects I am developing, and this leaves a number of lessons to be learned and improvements to be made. I will make sure deletions are not re-introduced to the project. 

Enjoy your sabbatical.